src/FEValues/common_values.jl

```diff
@@ -140,9 +164,30 @@ function function_value(fe_v::AbstractValues, q_point::Int, u::AbstractVector, d
     return val
 end
 
+"""
+    function_value(iv::InterfaceValues, q_point::Int, u::AbstractVector; use_element_a::Bool = true)
```

Comment:
I think I prefer
```suggestion
    function_value(iv::InterfaceValues, q_point::Int, u::AbstractVector; here::Bool)
```
i.e. `here` instead of `use_element_a`, and also to not use a default value but require the user to always ask for the side explicitly. (Same for the other functions but won't leave comments there).